{{cookiecutter.project_name}}/server/main_app/templates/main_app/index.html

```diff
@@ -9,6 +9,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link rel="stylesheet" href="{% static 'main_app/css/index.css' %}">
+
+    <link rel=“shortcut icon” type=“image/png” src=“{% static
+     "/media/logo.png" %}“>
+    <link rel=“shortcut icon” type=“image/jpg” src=“{% static
+     “/media/logo.jpg” %}“>
+    <link rel=“shortcut icon” type=“image/jpeg” src=“{% static
```

Comment:
You are using wrong `rel`. And wrong type. And you are missing `sizes`.
Here's an example how it should be: https://github.com/wemake-services/meta/blob/master/_includes/favicons.html#L11-L14